doc/ref_model/chapters/chapter07.md

```diff
@@ -317,11 +307,17 @@ The Operator’s responsibility is to not only make sure that security is includ
 Cloud Infrastructure operators must ensure that remote attestation methods are used to remotely verify the trust status of a given Cloud Infrastructure platform.  The basic concept is based on boot integrity measurements leveraging the Trusted Platform Module (TPM) built into the underlying hardware. Remote attestation can be provided as a service, and may be used by either the platform owner or a consumer/customer to verify that the platform has booted in a trusted manner. Practical implementations of the remote attestation service include the Open Cloud Integrity Tool (Open CIT).   Open CIT provides ‘Trust’ visibility of the Cloud Infrastructure and enables compliance in Cloud Datacenters by establishing the root of trust and builds the chain of trust across hardware, operating system, hypervisor, VM, and container.  It includes asset tagging for location and boundary control. The platform trust and asset tag attestation information is used by Orchestrators and/or Policy Compliance management to ensure workloads are launched on trusted and location/boundary compliant platforms. They provide the needed visibility and auditability of infrastructure in both public and private cloud environments.
 
 <a name="7.6.2"></a>
-### 7.6.2 Workload Image Scanning / Signing
+### 7.6.2 Workload Image
+
+Only workload images from trusted sources must be used. Secrets must be stored outside of the images.
+
+It is easy to tamper with workload images. It requires only a few seconds to insert some malware into a workload image file while it is being uploaded to an image database or being transferred from an image database to a compute node. To guard against this possibility, workload images can be cryptographically signed and verified during launch time. This can be achieved by setting up a signing authority and modifying the hypervisor configuration to verify an image’s signature before they are launched. 
```

Comment:
I think "can" is too weak here. In 7.5.1 we say that "Software must be code signed". Shouldn't we say "must" here?

Reply:
@ulikleber Yes I agree with you, I will change the text accordingly